hermes-2.cxx

```diff
@@ -2852,6 +2855,12 @@ int Hermes::rhs(BoutReal t) {
 
       ddt(Vort) -= FV::Div_par(Vort, 0.0, max_speed);
     }
+
+    if (phi_dissipation) {
+      // Adds dissipation term like in other equations, but depending on gradient of potential
+      // Note: Doesn't seem to need faster than sound speed
+      ddt(Vort) -= FV::Div_par(-phi, 0.0, sound_speed);
```

Comment:
Maybe I'm being dim, but why is this `Div_par`? That's a first derivative, not a diffusion term, right?

Reply:
It's being used for its gradient limiting diffusion term, rather than the advection. The net advection velocity is set to zero (second argument), but this operator contains a dissipation term:

https://github.com/boutproject/BOUT-dev/blob/master/include/bout/fv_ops.hxx#L295

On each side of the cell boundary, the quantity being advected (here `phi`) is reconstructed at the boundaries (the fR and fL values). There is a flow between cells proportional to the difference in these reconstructions. Because we use a second order method, the reconstruction is linear: If the profile is linear, then there is no flux, but if there are second derivatives, then there will be some flux.

Reply:
Some explanation of the method in this paper: https://iopscience.iop.org/article/10.1088/1361-6587/ab1321/meta

Reply:
Ah, thanks!